tests/test_unsupported.py

```diff
@@ -72,7 +72,6 @@ class Permissions(IntFlag):
     "percent_formatting": "a = '~ %s ~' % 'a'",  # https://github.com/adsharma/py2many/issues/176
     "tuple_destruct": "foo, (baz, qux) = 4, (5, 6); assert foo != (baz != qux)",  # https://github.com/adsharma/py2many/issues/155
     "float_1": "a = float(1)",  # https://github.com/adsharma/py2many/issues/129
-    "docstrings": "'''docstring'''\ndef main(): pass",  # https://github.com/adsharma/py2many/issues/7
```

Comment:
When removing anything from here, please add the same syntax into one of tests/cases/ to verify it is working correctly and prevent regressions.

If you also add docstrings to cases rect.py and *_enum.py , you should be able to remove some/all of those cases from the ignore list, because that is what golint is whining about.